hack/generate/update-pipelines.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+readonly version=$(yq read olm-catalog/serverless-operator/project.yaml project.version)
+# shellcheck disable=SC2038
+find .tekton/ \( -name "*-push.yaml" \) | xargs -I{} yq write --inplace "{}" "spec.params(name==additional-tags).value[0]" "${version}"
```

Comment:
Not really sure, but could we add a comment [here](https://github.com/openshift-knative/hack/blob/d63bd441112550420fc587cd4548252b215670df/pkg/prowgen/prowgen_konflux.go#L316), that the version should be on `Tags[0]`?
Just to reduce the probability that we break this in future

Reply:
I will do

Reply:
https://github.com/openshift-knative/hack/pull/339